auth-web/src/views/auth/MaintainBusinessView.vue

```diff
@@ -26,35 +26,45 @@
             @click="emitManageBusinesses()">
             Manage an Existing Business
           </v-btn>
-          <v-btn v-else large color="#fcba19" @click="login()" class="my-5">
-            Log in with BC Services Card
-          </v-btn>
-          <p>New to BC Registries? <a @click="emitAccountDialog()" class="create-account-link">
-            <u>Create a BC Registries Account</u></a>
-          </p>
-          <LearnMoreButton />
+          <template v-else>
+            <v-btn large color="#fcba19" @click="emitLogin()" class="my-5">
+              Log in with BC Services Card
+            </v-btn>
+            <p>New to BC Registries? <a @click="emitAccountDialog()" class="create-account-link">
+              <u>Create a BC Registries Account</u></a>
+            </p>
+          </template>
+          <learn-more-button
+           :redirect-url="learnMoreUrl"
+          />
         </div>
       </v-col>
       <!-- Image Column -->
       <v-col cols="12" md="6">
-        <v-img src="../../assets/img/Step4-Maintain-x1.png" aspect-ratio="1.2" contain></v-img>
+        <a :href="learnMoreUrl" target="_blank">
+          <v-img src="../../assets/img/Step4_Maintain_x2.png" aspect-ratio="1.2" contain></v-img>
+        </a>
       </v-col>
     </v-row>
   </v-container>
 </template>
 
 <script lang="ts">
 import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+import ContactUsTooltip from '@/components/auth/common/ContactUsTooltip.vue'
 import LearnMoreButton from '@/components/auth/common/LearnMoreButton.vue'
 import { Pages } from '@/util/constants'
 import { User } from '@/models/user'
 
 @Component({
   components: {
+    ContactUsTooltip,
     LearnMoreButton
   }
 })
 export default class MaintainBusinessView extends Vue {
+  private learnMoreUrl = 'https://www2.gov.bc.ca/assets/gov/employment-business-and-economic-development/business' +
```

Comment:
same here, ok to be a single string right?

Reply:
Definitely, will update!

Reply:
Updated in next commit!